lib/src/protos/git_store.proto

```diff
@@ -28,4 +28,5 @@ message Commit {
 
   bool is_open = 8 [deprecated = true];
   bool is_pruned = 9 [deprecated = true];
+  repeated string topics = 11;
```

Comment:
Can we not explicitly name it topics here? It should just be metadata as we want to support multiple use-cases with it, which topics are a subset of.